### ISSUE TITLE
Cr src returns error

### ISSUE THREAD

**Issue by grendal87** [wenyalintw/Google-Patents-Scraper#6]
When I attempt to run cd src main.py in cmd I get an error saying CD is not valid along with a few other errors. 

Is there perhaps a way to make a exe that is less tempermental?

**Comment by Frustrated87** [comment]
this can be closed. I was combining the cd src and main.py commands. 

I do however after sperating the commands run into a new error. Path, Line 1, in <module> from PyQt5.QtGui import Qicon, QDesktopServices ModuleNotFoundError: No module named 'PyQt5'

**Comment by bezeled** [comment]
I was also running into this - any solution?

**Comment by grendal87** [comment]
Yes first enter cd src command and switch into the directory that contains main.py. then the next command is to run main.py

The error fristrated87 ran into can be fixed by updating the moduals to the latest ones. Once it was all updated I was able to run it but couldn't get it to search or anything and have not progressed beyond this point.
